cmake/BipedalLocomotionFrameworkFindDependencies.cmake

```diff
@@ -177,6 +177,10 @@ framework_dependent_option(FRAMEWORK_COMPILE_Estimators
 framework_dependent_option(FRAMEWORK_COMPILE_Contact
   "Compile Contact libraries?" ON
   "FRAMEWORK_USE_manif" OFF)
+  
+framework_dependent_option(FRAMEWORK_COMPILE_Contacts
+  "Compile Contacts libraries?" ON
+  "FRAMEWORK_USE_manif" OFF)  
```

Comment:
Let's avoid having both ``FRAMEWORK_COMPILE_Contacts`` and ``FRAMEWORK_COMPILE_Contact``. It is super confusing.

Reply:
I intended to keep it similar, accounting for the case, where we will be porting the contents of `Contact` into `Contacts`.
If you see it problematic, would you have any suggestions for the naming?

Reply:
Maybe we can avoid having two options, and use only one to compile both libraries. This should be ok if we are planning to remove the old ``Contact``.

Reply:
I thought we were going to remove the old `Contact` library after migrating the necessary classes into the new library.

Reply:
Yes, I did not meant with this PR. I guess it is not a big deal if we use a single option to trigger the compilation of two libraries

Reply:
Ah ok. I understood and remembered only now that while I opened this PR, https://github.com/dic-iit/bipedal-locomotion-framework/pull/141 was still open. 

I will fix it.

Reply:
Fixed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/142/commits/576e5dc44b0a4ef36fd6c16d0c2e824add69d6ea